bamboo/unit_tests/test_unit_lbann2_reload.py

```diff
@@ -84,14 +85,26 @@ def skeleton_lbann2_reload(cluster, executables, dir_name, compiler_name,
         model_path='../../model_zoo/tests/model_lenet_mnist_lbann2ckpt.prototext',
         num_epochs=2,
         optimizer_name='sgd',
-        restart_dir=ckpt_dir,
+        restart_dir=ckpt_dir + "/trainer0/",
         output_file_name=output_file_name,
         error_file_name=error_file_name,
         weekly=weekly)
     return_code_ckpt_2 = os.system(command)
     tools.assert_success(return_code_ckpt_2, error_file_name)
-#    os.system('rm lbann2_ckpt/model0-epoch*')
-#    os.system('rm lbann2_nockpt/model0-epoch*')
+    # Delete the checkpoint from the second model it confuses the test
+    fileList = glob.glob('{no_ckpt}/trainer0/model1.*'.format(no_ckpt=no_ckpt_dir))
```

Comment:
This is not the right way to walk file trees in Python. But also, why not just `rmtree('{no_ckpt}/trainer0'.format(no_ckpt=no_ckpt_dir)`?

Reply:
Hmmm, why not.  The problem is that there is a file with the glob pattern and a set of directories with that name.  Also we don't want to delete everything under trainer0, just the model1 specific checkpoints.